packages/sdk/src/server/__tests__/server.test.ts

```diff
@@ -1,25 +1,65 @@
-import {
-  ServerClient, createClient,
-} from "../index";
 import fetchMock from "jest-fetch-mock";
 import { ClientCredentials } from "simple-oauth2";
 
-describe("ServerClient", () => {
-  let client: ServerClient;
+import {
+  BackendClient,
+  createBackendClient,
+  HTTPAuthType,
+} from "../index";
 
-  beforeEach(() => {
-    fetchMock.resetMocks();
+let client: BackendClient;
+let customDomainClient: BackendClient;
+const projectId = "proj_abc123";
+
+beforeEach(() => {
+  // @ts-ignore
```

Comment:
Why the `@ts-ignore`?

Reply:
Because `ClientCredentials` is now a mocked class but the Typescript compiler thinks it's the original one. I spent some time with [`jest-mock`](https://www.npmjs.com/package/jest-mock) and also playing around with the `ts-jest` config to no avail. Based on the material I reviewed it's possible to do it properly but IDK how much time it'd take. Can definitely keep looking into it though.

Reply:
no that makes sense, I was just curious but seems like the best approach is to keep for now